src/main/java/com/quirko/app/GameController.java

```diff
@@ -41,7 +62,12 @@ public DownData onDownEvent(MoveEvent event) {
                 board.getScore().add(1);
             }
         }
-        return new DownData(clearRow, board.getViewData());
+
+        if(SimpleBoard.colorCode!=0)
+        return new DownData(clearRow, board.getViewData(), clearColor);

```

Comment:
fix indentation

and enclose if and else body inside parenthesis

Reply:
Düzelttim.